Python 3.13.11, 3.14.2 regression in test_override_configured_linewrap_with_a_value_of_23

### Diagnostic output

```
jrnl: v4.2.1
Python: 3.13.11 (main, Dec  5 2025, 00:00:00) [GCC 15.2.1 20251111 (Red Hat 15.2.1-4)]
OS: Linux 6.17.12-300.fc43.x86_64
```

### Current Behavior

Since the system Python was updated from 3.13.9 to 3.13.11 in Fedora 42 (and since the system Python was updated from 3.14.0 to 3.14.2 in Fedora 43 and 44/Rawhide, although I know that `jrnl` still does not officially support Python 3.14), `test_override_configured_linewrap_with_a_value_of_23` fails due to a small difference in whitespace.

### Expected Behavior

All tests pass.

### Repro Steps

```
$ git clone https://github.com/jrnl-org/jrnl.git
$ cd jrnl
$ python3.13 --version
Python 3.13.11
$ tox -e py313
```

### Debug output

```
py313: install_deps> python -I -m pip install 'parse-type>=0.6.0' 'pytest-bdd>=8.0' 'pytest-xdist>=2.5.0' 'pytest>=8.1' 'toml>=0.10'
.pkg: _optional_hooks> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
.pkg: get_requires_for_build_sdist> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
.pkg: get_requires_for_build_wheel> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
.pkg: prepare_metadata_for_build_wheel> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
.pkg: build_sdist> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
py313: install_package_deps> python -I -m pip install 'colorama>=0.4' 'cryptography>=3.0' 'keyring>=21.0' 'parsedatetime>=2.6' 'python-dateutil<3.0,>=2.8' 'pyxdg>=0.27.0' 'rich<14.2.0,>=14
.1.0' 'ruamel.yaml.clib>=0.2.12' 'ruamel.yaml>=0.17.22' 'tzlocal>=4.0'
py313: install_package> python -I -m pip install --force-reinstall --no-deps /home/ben/src/forks/jrnl/.tox/.tmp/package/2/jrnl-4.2.1.tar.gz
py313: commands[0]> pytest
=================================================================================== test session starts ====================================================================================
platform linux -- Python 3.13.11, pytest-9.0.2, pluggy-1.6.0
cachedir: .tox/py313/.pytest_cache
rootdir: /home/ben/src/forks/jrnl
configfile: pyproject.toml
plugins: xdist-3.8.0, bdd-8.1.0
16 workers [685 items]    
ss.................................................s.............sss................................................................................................................ [ 26%]
.............s.................ssss.....................................................................................................F........................................... [ 52%]
.................................................................................................................................................................................... [ 78%]
........................................................................ss......s........s...s........ss.s.s.ssss.ssssssssssss.ss.s....s.........                                    [100%]
========================================================================================= FAILURES =========================================================================================
___________________________________________________________________ test_override_configured_linewrap_with_a_value_of_23 ___________________________________________________________________
[gw13] linux -- Python 3.13.11 /home/ben/src/forks/jrnl/.tox/py313/bin/python
Traceback (most recent call last):
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/runner.py", line 353, in from_call
    result: TResult | None = func()
                             ~~~~^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/runner.py", line 245, in <lambda>
    lambda: runtest_hook(item=item, **kwds),
            ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_hooks.py", line 512, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 167, in _multicall
    raise exception
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
    ~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/logging.py", line 850, in pytest_runtest_call
    yield
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
    ~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/capture.py", line 900, in pytest_runtest_call
    return (yield)
            ^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
    ~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/skipping.py", line 268, in pytest_runtest_call
    return (yield)
            ^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 121, in _multicall
    res = hook_impl.function(*args)
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/runner.py", line 179, in pytest_runtest_call
    item.runtest()
    ~~~~~~~~~~~~^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/python.py", line 1720, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_hooks.py", line 512, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 167, in _multicall
    raise exception
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pluggy/_callers.py", line 121, in _multicall
    res = hook_impl.function(*args)
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/python.py", line 166, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pytest_bdd/scenario.py", line 299, in scenario_wrapper
    _execute_scenario(feature, scenario, request)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pytest_bdd/scenario.py", line 274, in _execute_scenario
    _execute_step_function(request, scenario, step, step_func_context)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/pytest_bdd/scenario.py", line 240, in _execute_step_function
    return_value = call_fixture_func(fixturefunc=context.step_func, request=request, kwargs=kwargs)
  File "/home/ben/src/forks/jrnl/.tox/py313/lib/python3.13/site-packages/_pytest/fixtures.py", line 915, in call_fixture_func
    fixture_result = fixturefunc(**kwargs)
  File "/home/ben/src/forks/jrnl/tests/lib/then_steps.py", line 99, in output_should_be_docstring
    output_should_be(docstring, cli_run)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/ben/src/forks/jrnl/tests/lib/then_steps.py", line 106, in output_should_be
    assert actual == expected
AssertionError: assert '┎─────╮2013-...────────────┘' == '┎─────╮2013-...────────────┘'

  Skipping 47 identical leading characters in diff, use -v to show
  Skipping 72 identical trailing characters in diff, use -v to show

  - ┃ fir st  ent ry.     │
  ?         -
  + ┃ fir st ent ry.      │...

  ...Full output truncated (11 lines hidden), use '-vv' to show
================================================================================= short test summary info ==================================================================================
FAILED tests/bdd/test_features.py::test_override_configured_linewrap_with_a_value_of_23 - AssertionError: assert '┎─────╮2013-...────────────┘' == '┎─────╮2013-...────────────┘'
======================================================================== 1 failed, 644 passed, 40 skipped in 1.93s =========================================================================
py313: exit 1 (2.37 seconds) /home/ben/src/forks/jrnl> pytest pid=886008
.pkg: _exit> python /usr/lib/python3.14/site-packages/pyproject_api/_backend.py True poetry.core.masonry.api
  py313: FAIL code 1 (7.82=setup[5.45]+cmd[2.37] seconds)
  evaluation failed :( (7.85 seconds)
```

### Other Information

_No response_

--- a/tests/bdd/features/override.feature
+++ b/tests/bdd/features/override.feature
@@ -28,18 +28,22 @@ Feature: Implementing Runtime Overrides for Select Configuration Keys
         Given we use the config "simple.yaml"
         And we use the password "test" if prompted
         When we run "jrnl  -2 --config-override linewrap 23 --format fancy"
-        Then the output should be
+        Then the standard output should contain
             """
             ┎─────╮2013-06-09 15:39
             ┃ My  ╘═══════════════╕
-            ┃ fir st  ent ry.     │
+            """
+        And the standard output should contain
+            """
             ┠╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
             ┃ Everything is       │
             ┃ alright             │
             ┖─────────────────────┘
             ┎─────╮2013-06-10 15:40
             ┃ Lif ╘═══════════════╕
-            ┃ e is  goo d.        │
+            """
+        And the standard output should contain
+            """
             ┠╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
             ┃ But I'm better.     │
             ┖─────────────────────┘

